packages/teleport/src/services/joinToken/joinToken.ts

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import api from 'teleport/services/api';
+import cfg from 'teleport/config';
+import makeJoinToken from './makeJoinToken';
+import { JoinToken } from './types';
+
+const service = {
```

Comment:
we are moving away from global service, and prefer using classes to get one instance of it (makes testing easier too), look at other services where we define it as a class

Reply:
Thanks, I was wondering why some services are classes and others don't